test/models/UploadStatusSpec.scala

```diff
@@ -85,7 +85,7 @@ class UploadStatusSpec extends UnitSpec {
         val expectedUrl = "downloadUrl"
         val noOfRows = 2
         val expectedJson =
-          s"""{"reference":"ref1","fileName":"$expectedName","downloadUrl":"$expectedUrl","_type":"UploadedSuccessfully","noOfRows":$noOfRows}"""
+          s"""{"reference":"ref1","fileName":"$expectedName","downloadUrl":"$expectedUrl","noOfRows":$noOfRows,"_type":"UploadedSuccessfully"}"""
```

Comment:
why is this new field `noOfRows` coming up , have you changed any case classes ?

Reply:
if its due to upgrade of those upscan libs, then its fine, no issues

Reply:
not a new field just a change on the order of the json being returned because of the upgrade

Reply:
ah ok, i see it now.